### ISSUE TITLE
Support bootstrap v4 beta changes

### ISSUE THREAD

**Issue by eamodio** [tochoromero/aurelia-bootstrap#33]
Now that bootstrap v4 beta is finally out, it would be great if this could align with it :)

**Comment by sschocke** [comment]
Yes, aligning would be nice, considering the typeahead component does not even work anymore in bootstrap v4 beta

**Comment by tochoromero** [comment]
Yes, I need to sit down and upgrade to Bootstrap 4 beta.
Now that Bootstrap 4 is in Beta, and is more or less production ready (I have been using it for my projects for a while now) I will drop support for Bootstrap 3. I just don't have the time to support both versions.

**Comment by thejaff2** [comment]
Im currently not getting typehead working with bootstrap v4 beta, is this a known bug or am I doing something wrong? (Downgrading to v3.3.7 and NOT setting config.options.version = 4 is working for me)

**Comment by lukpep** [comment]
@thejaff2 - it's known - https://github.com/tochoromero/aurelia-bootstrap/issues/40
same for me

**Comment by stuartbale** [comment]
Bootstrap v4 is now at version 4.1.1
(see https://getbootstrap.com/)
Is it likely that you will update this project, or is it now abandoned?
